lib/rules/no-useless-token-range.ts

```diff
@@ -51,33 +58,35 @@ const rule = {
         arg.properties.length >= 2 ||
         (arg.properties.length === 1 &&
           (getKeyName(arg.properties[0]) !== 'includeComments' ||
-            arg.properties[0].value.type !== 'Literal'))
+            (arg.properties[0] as Property).value.type !== 'Literal'))
       );
     }
 
+    function isMemberExpression(node: Node): node is MemberExpression {
```

Comment:
I believe we can avoid simple type-checking functions like this. Inlining the type check should work fine for narrowing the type.

Reply:
Updated.  This was just for convenience, since there were so many.